### ISSUE TITLE
Error since version 0.0.19 (ConcurrentModificationException)

### ISSUE THREAD

**Issue by gerhardj-b** [kbuntrock/openapi-maven-plugin#139]
Hi, the recent version of the plugin (0.0.19) on a Spring Boot 3.2.3 project leads to an error on our code.

Running `mvn compile -X` leads to the attached stacktrace.

[stacktrace.txt](https://github.com/kbuntrock/openapi-maven-plugin/files/15480296/stacktrace.txt)

Due to company policies, I cannot publish the source that leads to the error, but in case needed, I can try to come up with a minimal example to reproduce this.

Best regards, Gerhard

**Comment by gerhardj-b** [comment]
Saving the value before the forEach fixes the issue for me, e.g.:

```patch
diff --git a/openapi-maven-plugin/src/main/java/io/github/kbuntrock/configuration/library/reader/SpringMvcReader.java b/openapi-maven-plugin/src/main/java/io/github/kbuntrock/configuration/library/reader/SpringMvcReader.java
--- a/openapi-maven-plugin/src/main/java/io/github/kbuntrock/configuration/library/reader/SpringMvcReader.java	(revision 0481290ac4f7803e5af42ec7452a5fa60cbeeee1)
+++ b/openapi-maven-plugin/src/main/java/io/github/kbuntrock/configuration/library/reader/SpringMvcReader.java	(date 1716915395886)
@@ -222,7 +222,8 @@
 
 		// Last case, some Dto fields can be binded to QueryParams : http://dolszewski.com/spring/how-to-bind-requestparam-to-object/
 		// Since this functionality is not well documented, it can be for now a subset of the complete functionality
-		parameters.values().stream().filter(x -> x.getLocation() == null && parameterObjectBindableToQueryParams(x)).forEach(paramObj -> {
+		List<ParameterObject> intermediateResult = parameters.values().stream().filter(x -> x.getLocation() == null && parameterObjectBindableToQueryParams(x)).collect(Collectors.toList());
+		intermediateResult.forEach(paramObj -> {
 			bindDtoToQueryParams(parameters, paramObj);
 		});
```

**Comment by kbuntrock** [comment]
Thank you for the report. Should be fixed #140 is merged.

**Comment by kbuntrock** [comment]
Fixed by #140

Thank you for the bug report @gerhardj-b 🙏
